### DISCUSSION TITLE
Map download queue

### DISCUSSION THREAD

**Discussion by ioxusEosa** [ppy/osu#27749] [Ideas]
I think it'd be pretty nice to be able to set how many downloads can occur simultaneously or just be able to queue downloads.

**Comment by Walavouchey** [comment]
is this just to limit bandwidth? such things can be achieved with external software

if it's because you find yourself downloading too many things by accident, you can cancel downloads from the notification

if this happens, a limit of simultaneous downloads most likely wouldn't be configurable
